crates/noirc_abi/src/lib.rs

```diff
@@ -97,6 +116,60 @@ pub enum Sign {
 }
 
 impl AbiType {
+    // TODO: Add `Context` argument for resolving fully qualified struct paths
```

Comment:
Can you link issue here before we merge?

Reply:
This is solved and removed in #2374 so no need to track it